packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -70,7 +60,7 @@ export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   }, []);
 
   return (
-    <EditorContainer bg="black" as="main" isIFrame={isIFrame}>
+    <EditorContainer bg="black" height="25rem" width="43rem" {...rest}>
```

Comment:
here we can do:

    <EditorContainer
      bg="black"
      style={{ height: '25rem', width: '43rem', ...style }}
    >
    
if we still really need rest we can pass it in too, but i think this makes it more explicit that we are overriding styles here. 

mostly following this pattern that kent dodds outlines: https://egghead.io/lessons/react-style-react-components-with-classname-and-in-line-styles

Reply:
also fixes some issues where storybook is collapsing in this implementation, so we'll get a full:

<img width="1437" alt="Screen Shot 2022-02-04 at 12 16 37 AM" src="https://user-images.githubusercontent.com/4481281/152476518-3606846b-832f-42cd-8a3c-75bbebc599f6.png">

Reply:
checking in storybook too, and the approach suggested accepts style overrides really nicely